src/Symfony/Component/Config/Tests/Resource/FileExistenceResourceTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Config\Tests\Resource;
+
+use Symfony\Component\Config\Resource\FileExistenceResource;
+
+class FileExistenceResourceTest extends \PHPUnit_Framework_TestCase
+{
+    protected $resource;
+    protected $file;
+    protected $time;
+
+    protected function setUp()
+    {
+        $this->file = realpath(sys_get_temp_dir()).'/tmp.xml';
+        $this->time = time();
+        $this->resource = new FileExistenceResource($this->file);
+    }
+
+    protected function tearDown()
+    {
+        if (file_exists($this->file)) {
+            unlink($this->file);
+        }
+    }
+
+    public function testToString()
+    {
+        $this->assertSame($this->file, (string) $this->resource);
+    }
+
+    public function testGetResource()
+    {
+        $this->assertSame($this->file, $this->resource->getResource(), '->getResource() returns the path to the resource');
+    }
+
+    public function testIsFreshWithExistingResource()
```

Comment:
See https://gist.github.com/mpdude/a123e0233dba164ef896, that's how I would do it.